### ISSUE TITLE
Please don't remove platform icons during sales

### ISSUE THREAD

**Issue by kamjau** [ValveSoftware/steam-for-linux#4223]
Most games aren't available for Linux, so I just end up assuming everything on the front page isn't when there's no icons to tell otherwise. I know I can hover to show the icons, but that's time consuming and doesn't always work.

**Comment by not83** [comment]
Is this an issue still?

**Comment by kamjau** [comment]
Yes. It's a bigger issue during sales, but there's games with no platform icons on the front page right now too. The weekend deal, new on steam, recently updated, recommended for you, curated recommendations, and specials boxes don't have any platform information for their games. I think most of those used to have them. During the big sales none of the games on the front page have platform icons.
